src/types/FantasyStats.ts

```diff
@@ -0,0 +1,33 @@
+
+export enum StatProductionType {
+    ACTUAL,
+    PROJECTED
+}
+
+export type Stat = {
+    name: string,
+    value: number,
+    enabled: boolean
+}
+
+export type StatProduction = {
+    // the intention is that this is extended down the road with more stats
+    PTS? : number,      // points
+    REB? : number,      // rebounds
+    AST? : number,      // assists
+    STL? : number,      // steals
+    BLK? : number,      // blocks
+    TOV? : number       // turnovers
```

Comment:
if these are expected to change, then maybe this type should just extend a Map<StatType, number>.

where StatType is an enum or "as const" of your list of stats. That way, you don't need to update this type and the type below every time a new stat is added.

Reply:
Good point, will make that change.